src/test/kotlin/net/maizegenetics/phgv2/cli/MergeHvcfsTest.kt

```diff
@@ -0,0 +1,179 @@
+package net.maizegenetics.phgv2.cli
+
+import com.github.ajalt.clikt.testing.test
+import net.maizegenetics.phgv2.utils.getChecksum
+import org.apache.logging.log4j.LogManager
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import java.io.File
+import kotlin.test.assertEquals
+
```

Comment:
If you add test cases that run the clikt command without the required parameters you should get additional code coveragte.

Reply:
oh - maybe not as you don't have a validate clause.  Coverage will be the same.